src/app/components/wrapper-components/GameWrapper.ts

```diff
@@ -89,16 +88,16 @@ export abstract class GameWrapper<P extends Comparable> {
         this.gameComponent = componentRef.instance;
 
         this.gameComponent.chooseMove = // so that when the game component do a move
-            (m: Move, s: GameState, scores?: [number, number]): Promise<MGPValidation> => {
-                return this.receiveValidMove(m, s, scores);
+            (m: Move): Promise<MGPValidation> => {
+                return this.receiveValidMove(m);
```

Comment:
```
// so that when the game component do a move
this.gameComponent.chooseMove = (m: Move): Promise<MGPValidation> => {
    return this.receiveValidMove(m);
}
```
?